src/main/java/org/prebid/server/bidder/openx/proto/OpenxVideoExt.java

```diff
@@ -0,0 +1,15 @@
+package org.prebid.server.bidder.openx.proto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+@Builder
+public class OpenxVideoExt {
+
+    @JsonProperty("rewarded")
```

Comment:
Minor. We don't need annotation here, since the field name will not be changed after marshaling to JSON.